node/payment-links.js

```diff
@@ -0,0 +1,182 @@
+const https = require('https');
+const Buffer = require('buffer').Buffer;
+
+const getPaymentLinkList = (api_key, antifraudMetadata) => {
+    const antifraudMetadataBase64 = Buffer.from(JSON.stringify(antifraudMetadata)).toString('base64');
+    const options = {
+        method: 'GET',
+        headers: {
+            'X-Api-Client-Key': api_key,
+            'X-Cash-Anti-Fraud-Metadata': antifraudMetadataBase64,
+            'Content-Type': 'application/json'
+        },
+    };
+    const url = `https://api-v2.sandbox.holacash.mx/v2/payment_link?limit=1`;
+    return new Promise((resolve, reject) => {
+        const request = https.request(url, options, (response) => {
+            let data = ''
+            response.on('data', chunk => data += chunk);
+            response.on('end', () => {
+                const parsedData = JSON.parse(data);
+                if ([200, 201].includes(response.statusCode)) {
+                    resolve(parsedData);
+                    return;
+                }
+                reject(parsedData);
+            });
+        }).on('error', err => reject(err));
+        request.end();
+    });
+};
+
+const createPaymentLink = (body, api_key, antifraudMetadata) => {
+    const antifraudMetadataBase64 = Buffer.from(JSON.stringify(antifraudMetadata)).toString('base64');
+    const options = {
+        method: 'POST',
+        headers: {
+            'X-Api-Client-Key': api_key,
+            'X-Cash-Anti-Fraud-Metadata': antifraudMetadataBase64,
+            'Content-Type': 'application/json'
+        },
+    };
+    const url = 'https://api-v2.sandbox.holacash.mx/v2/payment_link';
+    return new Promise((resolve, reject) => {
+        const request = https.request(url, options, (response) => {
+            let data = ''
+            response.on('data', chunk => data += chunk);
+            response.on('end', () => {
+                const parsedData = JSON.parse(data);
+                if ([200, 201].includes(response.statusCode)) {
+                    resolve(parsedData);
+                    return;
+                }
+                reject(parsedData);
+            });
+        }).on('error', err => reject(err));
+        request.write(JSON.stringify(body));
+        request.end();
+    });
+};
+
+const getPaymentLinkDetail = (payment_link_id, api_key, antifraudMetadata) => {
```

Comment:
@sahil-holacash call it only `getPaymentLink`.

Reply:
updated